R/hover_reload_button.R

```diff
@@ -0,0 +1,70 @@
+#' Reload button with button and icon animations
+#'
+#' Animate a reloadButton and it's icon using
+#' [Hover.css](https://github.com/IanLunn/Hover)
```

Comment:
Since `reloadButton` isn't an actual button type from `shiny`, let's reword:
> Animate a reload button and it's icon using [Hover.css](https://github.com/IanLunn/Hover). Note that a reload button is just a `shiny::actionButton` with onClick behavior to reload or refresh a web browser.